package.json

```diff
@@ -58,12 +58,12 @@
     "@emotion/eslint-plugin": "^11.2.0",
     "@emotion/jest": "^11.2.1",
     "@emotion/snapshot-serializer": "^0.8.2",
-    "@storybook/addon-a11y": "^6.0.26",
-    "@storybook/addon-knobs": "^6.0.26",
-    "@storybook/addon-links": "^6.0.26",
+    "@storybook/addon-a11y": "^6.1.21",
+    "@storybook/addon-knobs": "^6.1.21",
+    "@storybook/addon-links": "^6.1.21",
     "@storybook/addon-notes": "^5.3.21",
```

Comment:
`addon-notes` has been deprecated, and I intend to follow up by looking into its replacement: `addon-docs`, especially as we're planning on improving our component documentation.